tests/conftest.py

```diff
@@ -83,3 +84,44 @@ def engine(request, database, spark_database):
         return QueryEngineFixture(name, spark_database, SparkQueryEngine)
     else:
         assert False
+
+
+@pytest.fixture
+def bool_series():
+    return lambda: clinical_events.exists_for_patient()
+
+
+@pytest.fixture
+def code_series():
+    return (
+        lambda: clinical_events.sort_by(clinical_events.code)
+        .first_for_patient()
+        .select_column(clinical_events.code)
+    )
+
+
+@pytest.fixture
+def date_series():
+    return (
+        lambda: clinical_events.sort_by(clinical_events.date)
+        .first_for_patient()
+        .select_column(clinical_events.date)
+    )
+
+
+@pytest.fixture
+def int_series():
+    return (
+        lambda: clinical_events.sort_by(clinical_events.value)
+        .first_for_patient()
+        .select_column(clinical_events.value)
+    )
+
+
+@pytest.fixture
+def patient_series():
+    return (
+        lambda: clinical_events.sort_by(clinical_events.code)
+        .first_for_patient()
+        .select_column(clinical_events.code)
+    )
```

Comment:
Is there any reason these all return lambdas?

Reply:
The fixtures all need to return a callable so we can get a new Series when we call them.

Reply:
Why not make it a function. I don't think we're gaining anything from this being a fixture.